var/spack/repos/builtin/packages/py-py4j/package.py

```diff
@@ -13,8 +13,12 @@ class PyPy4j(PythonPackage):
     homepage = "https://www.py4j.org/"
     url = "https://pypi.io/packages/source/p/py4j/py4j-0.10.4.zip"
 
+    version('0.10.7', sha256='721189616b3a7d28212dfb2e7c6a1dd5147b03105f1fc37ff2432acd0e863fa5')
     version('0.10.6', sha256='d3e7ac7c2171c290eba87e70aa5095b7eb6d6ad34789c007c88d550d9f575083')
     version('0.10.4', sha256='406fbfdbcbbb398739f61fafd25724670a405a668eb08c1721d832eadce06aae')
     version('0.10.3', sha256='f4570108ad014dd52a65c2288418e31cb8227b5ecc39ad7fc7fe98314f7a26f2')
 
     depends_on('py-setuptools', type='build')
+    depends_on('py-flake8@2.4.1:', type='test')
+    depends_on('py-tox@2.1.1:', type='test')
+    depends_on('py-nose@1.3.7:', type='test')
```

Comment:
Where did you find these? The nose dependency makes sense for running `python setup.py test`, but tox and flake8 are usually developer tools not needed for unit testing.